src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -2,16 +2,32 @@ import { pxToRem } from '../../../../lib'
 import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
 import { IListItemProps } from '../../../../components/List/ListItem'
 
+const hoverStyle = variables => ({
```

Comment:
do we see these hover styles being generically applied across Teams theme to all List items? Frankly, would doubt that (but please, feel free to object and provide the points where I am wrong). To me it seems to be quite reasonable to expect that at some places in Teams the ListItem would not require to hide header and content media on hover. 

If this assumption is correct, shouldn't we introduce component wrapper for `ListItem` instead (`TeamsMessageListItem`) - and provide these styles to be merged on top of the ones of `ListItem`?

Reply:
We checked the chat list and calling roster list behavior. This seems to be the default styles, so will keep it like this and in case we want a different style (Teams/Channels list), we would add a wrapper there.